index.ts

```
@@ -182,11 +178,27 @@ const main = async () => {
       logger.info(`Server listening on port ${appConfig.APP_PORT}`);
     });
-  } else {
-    logger.info(
-      'Server not started in test environment (tests will manage their own server instances if needed).',
-    );
-  }
+
+    logger.info('Server is ready.');
+
+    // Run startup tasks after server is ready
+    if(!process.env.CI) {
```

ESLint

[error] Insert `·`

(prettier/prettier)